src/test/java/visualizations/LineChartTest.java

```diff
@@ -0,0 +1,41 @@
+package visualizations;
+
+import dataloader.CsvDataLoader;
+import org.junit.Test;
+import tech.tablesaw.api.Table;
+
+import java.io.File;
+import java.io.IOException;
+
+public class LineChartTest {
+
+    /**
+     * Test Scenario: The user provides a data set
+     * and wants to plot a line chart.
+     * Expected Output: The chart is plotted correctly.
+     */
+    @Test
+    public void testPlotLine() throws IOException {
```

Comment:
Tests should assert something right??? One way is to save a reference image of the data you are plotting and then create the plot again and compare the two pixel-wise...Imagine the situation where the unit tests are run on a remote server. There you don't have a browser to view the results....Currently, the plotting functions are just wrappers to Tablesaw functionality thus I would say we don't need these tests. Instead move them to the examples better....what do you think?

Reply:
I agree and will change these test to examples. Though, check out tablesaw's api and you'll see he tests the plotting the same way I did. Perhaps the examples should be more descriptive and have some more implementations than just plotting.

Reply:
created the examples. pls check them out.

Reply:
ok